rpm/htcondor-ce.spec

```diff
@@ -349,6 +361,15 @@ fi
 %config(noreplace) %{_sysconfdir}/condor/config.d/99-ce-bdii.conf
 %endif
 
+
+%if ! 0%{?osg}
+%files apelscripts
+%config(noreplace) %{_datadir}/apel/condor_ce_blah.sh
+%config(noreplace) %{_datadir}/apel/condor_ce_batch.sh
+%config(noreplace) %{_datadir}/apel/accountingRun.sh
```

Comment:
I think we should do as much as we can to separate the config bits from the scripts so these should not be marked as config and should live in `%{_bindir}`

Reply:
See discussion below